mbutil/util.py

```diff
@@ -177,12 +177,10 @@ def disk_to_mbtiles(directory_path, mbtiles_file, **kwargs):
     cur = con.cursor()
     optimize_connection(cur)
     mbtiles_setup(cur)
-    #~ image_format = 'png'
     image_format = kwargs.get('format', 'png')
 
     try:
         metadata = json.load(open(os.path.join(directory_path, 'metadata.json'), 'r'))
-        image_format = kwargs.get('format')
```

Comment:
@andrewharvey I'm wondering if the original intent in this line is to look up the image format in the metadata file, in which case we should make this `image_format = metadata.get('format', image_format)` instead of deleting it. That means: try to get the format from metadata.json and if we can't, fall back on the command line value. Does that make sense?

Reply:
That makes sense, it's what I originally implemented in https://github.com/mapbox/mbutil/compare/master...andrewharvey:patch-1 but wasn't sure how we actually wanted it to work.

Your assumptions in your comment make sense to me. Use metadata.json then fallback to command line arguments.

I've opened #94 which implements that, replacing this.